app/blog/[slug]/page.tsx

```diff
@@ -12,12 +14,32 @@ async function fetchPost(slug) {
     date: post.createdAt,
     title: post.title,
     content: mdxSource,
-    slug,
   };
 }
 
+export const revalidate = 60 * 30;
```

Comment:
Can you change the revalidate to 30 secs instead of 30 min so we can test this quicker?